### PR TITLE
Clarify enhancements lead time requirements

### PR DESCRIPTION
**Pull request by JamesLaverack**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

The time requirements documented for Enhancements Lead are... low. During my time as Enhancements Lead for Kubernetes v1.22, I logged the time I spent on upstream work and came with the following:

- Before the release (w/c 12th April 2021): 3 hours
- Week 0 (w/c 19th April 2021): 6.5 hours
- Week 1 (w/c 26th April 2021): 15 hours
- Week 2 (w/c 3rd May 2021): 10 hours
- Week 3 (w/c 10th May 2021, enhancements freeze week): 25.5 hours
- Week 4 (w/c 17th May 2021, to date): 3 hours

Given this data, and [discussions with former Enhancements Leads](https://kubernetes.slack.com/archives/C2C40FMNF/p1621551220054200), I'm increasing the documented time requirements. I'm also clarifying the requirements for shadows.

### CODE REVIEW

**Review by salaxander — APPROVED** [review]

File: release-team/role-handbooks/enhancements/README.md [hunk 1]
```diff
@@ -43,10 +43,17 @@ Helpful characteristics of an Enhancements Lead include:
...
+
+Enhancement is one of the most time-intensive areas of the release team, and especially so during the early parts of the release. An Enhancements Lead can expect to spend:
+- Beginning of the cycle through enhancement freeze: ~8–15 hours a week
+- Week of enhancements freeze: 20+ hours
+- Enhancement Freeze through Code Freeze: ~4–7 hours a week
+- Code Freeze through Release Day: ~1–4 hours a week
+
+Note that Enahncements Lead in particular will need to do work **during the week** during the early release, and will need to be available at least daily.
+
+Enhancements shadows have much lower time committments, but can still expect to spend 5+ hours a week during the early release, and ~1–4 hours a week after enhancements freeze. Unlike Enhancements Lead, shadows can expect to do their work largely on weekends if they desire.
```
Comment (annajung) [inline release-team/role-handbooks/enhancements/README.md:56]: This is where we can really set the shadow expectation for the enhancements shadows for the future cycles. Shadows should plan to spend more than min 5 hours/week especially at the beginning of the cycle. That will help grow a better enhancements team and a better successor for future cycles.

To me, we should expect 10+ hours. Leads are responsible for double-checking shadows work which requires more commitment, but shadows should also be taking ownership of the assigned tasks related to the enhancements. We should clearly put that expectation here.

**Comment by JamesLaverack** [comment]
Oh oops. I was absolutely not intending to merge this already, but I forgot to `/hold` it. 🤯 Just trying to figure out how to back out changes with Prow...

**Comment by salaxander** [comment]
@annajung sorry this merged so quick, didn't realize my approval would trigger a merge on this repo 😅

File: release-team/role-handbooks/enhancements/README.md [hunk 2]
```diff
@@ -116,7 +123,7 @@ It is important that this process be followed and documentation remain up-to-dat
 - Work with the Release Lead to introduce yourself, talk about release information, and relay information about opting into the release with SIG Leads.
 - Weekly Release meetings require updates of current status. Use the `Feature stats` tab to update the release team on counts of enhancements in good and bad progress.
 
-#### Collection Monitoring and Triage (Week 2-3)
+#### Collection Monitoring and Triage (Week 2)
```
Comment (annajung) [inline release-team/role-handbooks/enhancements/README.md:126]:
```suggestion
#### Collection Monitoring and Triage (Week 1-2)
```

**Comment by annajung** [comment]
no worries @JamesLaverack @salaxander! :) In the future, we should put a hold on PR that requires multiple feedback.

**Comment by JamesLaverack** [comment]
Okay. I reverted it back out in https://github.com/kubernetes/sig-release/pull/1580 and re-opened it in https://github.com/kubernetes/sig-release/pull/1581. Please move discussion there. 😅

**Pull request merged**